libs/design-system/color-picker/src/lib/color-picker.stories.ts

```diff
@@ -16,7 +16,7 @@ const meta: Meta = {
       providers: [],
     }),
     moduleMetadata({
```

Comment:
Remove the entire `decorators` property. The component already imports the necessary directives, etc.